<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_binfo</class>
 <widget class="QDialog" name="Admin_binfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>288</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>481</width>
    <height>288</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>481</width>
    <height>288</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BookID</string>
   </property>
   <property name="buddy">
    <cstring>userlineEdithao</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BookName：</string>
   </property>
   <property name="buddy">
    <cstring>userlineEditming</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="userlineEdithao">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>193</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="apushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userlineEditzuo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>193</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="userlineEditming">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>193</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="userlineEditchu">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>193</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Publish</string>
   </property>
   <property name="buddy">
    <cstring>userlineEditchu</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
   <property name="buddy">
    <cstring>userlineEditzuo</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="dpushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>193</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>IT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>物理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>宗教</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>政治</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>法律</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>历史</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>生物</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>工业</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>农业</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>医药</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>食品</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>军事</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>化学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小说</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>戏曲</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BookItem</string>
   </property>
   <property name="buddy">
    <cstring>comboBox</cstring>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userlineEdithao</tabstop>
  <tabstop>userlineEditming</tabstop>
  <tabstop>userlineEditzuo</tabstop>
  <tabstop>userlineEditchu</tabstop>
  <tabstop>spushButton</tabstop>
  <tabstop>apushButton</tabstop>
  <tabstop>dpushButton</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
